panvimdoc.sh

```diff
@@ -138,6 +138,7 @@ ARGS=(
 # Add an additional lua filter if demojify is true
 if [[ ${DEMOJIFY:-false} == "true" ]]; then
     ARGS+=(
+        "--data-dir=$SCRIPTS_DIR/../lib"
```

Comment:
I think this change is not required because again in the Dockerfile we copy the `Demojify` file to a lua lib folder:

https://github.com/kdheepak/panvimdoc/blob/0bad620fb9e2e6ef14c16e80579a487e70089b10/Dockerfile#L8

Reply:
I guess there's no harm in doing this since it just adds an additional `--data-dir` for pandoc.